deepchecks/nlp/checks/data_integrity/text_duplicates.py

```diff
@@ -0,0 +1,137 @@
+        self.max_text_length_for_display = max_text_length_for_display
+
+    @property
+    def _text_normalization_kwargs(self):
+        return {
+            'ignore_case': self.ignore_case,
+            'ignore_whitespace': self.ignore_whitespace,
+            'normalize_uni': self.normalize_unicode,
+            'remove_punct': self.remove_punctuation,
+            'remove_stops': self.remove_stopwords,
+        }
+
+    def _truncate_text(self, x: str) -> str:
+        return truncate_string(x, self.max_text_length_for_display)
+
+    def run_logic(self, context: Context, dataset_kind):
+        """Run check."""
+        dataset = context.get_data_by_kind(dataset_kind).sample(self.n_samples, random_state=self.random_state)
+        dataset = t.cast(TextData, dataset)
+        samples = dataset.text
+        sample_hashes = hash_samples(normalize_samples(samples, **self._text_normalization_kwargs))
+
+        df = pd.DataFrame({
+            'Text': samples,
+            'hash': sample_hashes,
+            'Sample ID': dataset.get_original_text_indexes()
+        })
+        grouped_samples = df.groupby(by=['hash'], dropna=False)
+        counted_samples = grouped_samples['Text'].size()
+        n_of_unique = len(counted_samples)
+        n_of_samples = df.shape[0]
+        percent_of_duplicates = 1 - n_of_unique / n_of_samples
+
+        counted_duplicates = counted_samples[counted_samples > 1]
+        duplicates_hashes = set(counted_duplicates.index)
+
+        result_df = df[df['hash'].isin(duplicates_hashes)]
+        result_df = result_df.rename(columns={'hash': 'Duplicate'})
+        duplicates_enumeration = to_ordional_enumeration(result_df['Duplicate'].to_list())
+        result_df['Duplicate'] = result_df['Duplicate'].apply(lambda x: duplicates_enumeration[x])
+        result_df = result_df.set_index(['Duplicate', 'Sample ID'])
+
+        result_value = {
+            'percent_of_duplicates': percent_of_duplicates,
+            'duplicates': result_df
+        }
+
+        if not (context.with_display and percent_of_duplicates > 0):
+            return CheckResult(value=result_value)
+
+        grouped_samples = df[df['hash'].isin(duplicates_hashes)].groupby(by=['hash'], dropna=False)
+        first_sample = grouped_samples['Text'].first()
+        instances = grouped_samples['Sample ID'].aggregate(lambda x: format_list(x.to_list()))
+
+        table = pd.DataFrame({
+            'Text': first_sample.apply(self._truncate_text),
+            'Instances': instances,
+            'Number of Samples': counted_duplicates
+        })
+        table = table.iloc[:self.n_to_show]
+        table = table.sort_values(by=['Number of Samples'], ascending=False)
+        table = table.set_index(['Instances', 'Number of Samples'])
+
+        return CheckResult(
+            value=result_value,
+            display=[
+                f'{format_percent(percent_of_duplicates)} of data samples are duplicates.',
```

Comment:
```suggestion
                f'{format_percent(percent_of_duplicates)} of data samples are duplicates.\n',
```

Reply:
no need, each string item in the `display` sequence will be wrapped with HTML `<p>` tag, 
so, they will be displayed on different lines